src/quo/components/wallet/account_overview/schema.cljs

```diff
@@ -0,0 +1,20 @@
+(ns quo.components.wallet.account-overview.schema)
+
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map
```

Comment:
Since we didn't yet arrive at a decision on open/close maps this is not critical, but since this component's props are simpler I'd suggest we close the map to keep it consistent with the rest of the schemas.

Reply:
This one is used in some places that receives data directly from backend, so a lot of extra props are passed to it. It's better to let it be open for now

Reply:
+1 for open. 

1. Backend passes props that the component might not need
2. Quo preview passes `@state` while calling the component, which may contain extra kv pairs.